packages/libpng.rb

```diff
@@ -4,39 +4,25 @@ class Libpng < Package
   description 'libpng is the official PNG reference library.'
   homepage 'http://libpng.org/pub/png/libpng.html'
   @_ver = '1.6.37'
-  version "#{@_ver}-1"
+  version "#{@_ver}+apng"
```

Comment:
It would be better to use a numeric value here because there might be future iterations.

Reply:
I was thinking the next iteration could be "#{@_ver}+apng-1"

Reply:
Normally, features should be included in the description, not the version.

Reply:
Clearly, we need a better way of describing build variants...